Fix egress mtls origination test (#14914)
istio/istio.io: content/en/docs/tasks/traffic-management/egress/egress-gateway-tls-origination/index.md

      namespace: mesh-external
      labels:
        run: my-nginx
      annotations

>         networking.istio.io/exportTo: "." # simulate a test service by not exporting outside this namespace

Reviewer: ```suggestion
        networking.istio.io/exportTo: "." # simulate an external service by not exporting outside this namespace
```
